docs/triaging/insights/taxonomy-insights-documentation.md

```diff
@@ -0,0 +1,57 @@
+# InstructLab Contribution Insights
+
+This module automatically analyzes the taxonomy of pull request contribution files, checking for potential code of conduct violations.
```

Comment:
It is not stated whether insights also analyzes knowledge documents. Can we state, in this document, how knowledge documents are analyzed or not?

Reply:
Analysis for the referenced markdown files is out of scope for this initial version, but it's in plan. I've made an explicit mention. Let me know if the text is clear enough.